src/cookie.rs

```diff
@@ -92,7 +92,159 @@ pub(crate) fn extract_response_cookies<'a>(
 
 /// A persistent cookie store that provides session support.
 #[derive(Default)]
-pub(crate) struct CookieStore(pub(crate) cookie_store::CookieStore);
+pub struct CookieStore(pub cookie_store::CookieStore);
+
+impl CookieStore {
```

Comment:
We should probably try to pick what methods are definitely needed at first, and only add new methods as the need arises. Which seem absolutely necessary?

Reply:
We could ignore some methods but there will always be someone needing them. And it's not hard to maintain the methods since they are aliases.